### PR TITLE
Update changelog and secpoll for 4.1.0.

### PR DESCRIPTION
**Pull request by aerique**

### CODE REVIEW

File: docs/changelog/4.1.rst [hunk 1]
```diff
@@ -1,6 +1,59 @@
 Changelogs for 4.1.x
 ====================
 
+.. changelog::
+  :version: 4.1.0
+  :released: 29th of November 2017
+
+  This is the first release in the 4.1.x train.
+
+  The full release notes can be read at: https://wordpress.com/post/blog.powerdns.com/2473
```
Comment (pieterlexis) [inline docs/changelog/4.1.rst]:
```rst
can be read `on the blog <https://wordpress.com/post/blog.powerdns.com/2473>`_
```

Also, the URL will be prettier (with the title in the url) afaik

File: docs/changelog/4.1.rst [hunk 2]
```diff
@@ -1,6 +1,59 @@
 Changelogs for 4.1.x
 ====================
 
+.. changelog::
+  :version: 4.1.0
+  :released: 29th of November 2017
+
+  This is the first release in the 4.1.x train.
```
Comment (pieterlexis) [inline docs/changelog/4.1.rst]: s/4.1.x/4.1/

File: docs/changelog/4.1.rst [hunk 3]
```diff
@@ -1,6 +1,59 @@
...
+  - Improved performance: 400% speedup in some scenarios,
+  - Crypto API: DNSSEC fully configurable via RESTful API,
+  - Improved documentation,
+  - Database related improvements,
+  - Enhanced tooling,
+  - Support for TCP Fast Open,
+  - Support for non-local bind,
+  - Our packages now ship with PKCS #11 support.
+
+  And, while we recognize this has an impact on many installations and may be painful, it was necessary to remove support for Recursor passthrough.  To read more about this please see the blog post mentioned above.
```
Comment (pieterlexis) [inline docs/changelog/4.1.rst]: Add a link to the docs here as well

```rst
or read the :doc:`migration guide <../../guides/recursion>`.
```

File: docs/changelog/4.1.rst [hunk 4]
```diff
@@ -1,6 +1,60 @@
...
+  - Crypto API: DNSSEC fully configurable via RESTful API,
+  - Improved documentation,
+  - Database related improvements,
+  - Enhanced tooling,
+  - Support for TCP Fast Open,
+  - Support for non-local bind,
+  - Support for Botan 2.x (and removal of support for Botan 1.10),
+  - Our packages now ship with PKCS #11 support.
+
+  And, while we recognize this has an impact on many installations and may be painful, it was necessary to remove support for Recursor passthrough.  To read more about this please see the blog post mentioned above or read the :doc:`migration guide <../../guides/recursion>`.
```
Comment (ahupowerdns) [inline docs/changelog/4.1.rst]: this paragraph falls out of thin air?

**Pull request merged**